sdv/metadata/single_table.py

```diff
@@ -135,6 +135,12 @@ def _validate_unexpected_kwargs(self, column_name, sdtype, **kwargs):
                 f"Invalid values '({unexpected_kwargs})' for {sdtype} column '{column_name}'.")
 
     def _validate_sdtype(self, sdtype):
+        if not isinstance(sdtype, str):
+            raise InvalidMetadataError(
+                f'Invalid sdtype : {sdtype} is not a string. Please use one of the '
```

Comment:
There should be no space between the column and `sdtype`: `Invalid sdtype:`. It's probably just a typo in the issue itself.